doc/man3/BIO_s_mem.pod

```diff
@@ -89,16 +120,6 @@ a buffering BIO to the chain can speed up the process.
 Calling BIO_set_mem_buf() on a BIO created with BIO_new_secmem() will
 give undefined results, including perhaps a program crash.
 
-Switching the memory BIO from read write to read only is not supported and
-can give undefined results including a program crash. There are two notable
-exceptions to the rule. The first one is to assign a static memory buffer
-immediately after BIO creation and set the BIO as read only.
-
-The other supported sequence is to start with read write BIO then temporarily
-switch it to read only and call BIO_reset() on the read only BIO immediately
-before switching it back to read write. Before the BIO is freed it must be
-switched back to the read write mode.
-
 Calling BIO_get_mem_ptr() on read only BIO will return a BUF_MEM that
```

Comment:
Why this deletion?

Reply:
Because it is incorrect. AFAICT from code inspection, it is actually not possible to switch from a memory BIO to a read only BIO. The only way to create a read only BIO is via `BIO_new_mem_buf`. I don't see any other way to do it.

Reply:
Duh. Right. I somehow missed that. I have resurrected this text.